### PR TITLE
Cleanup of the EventAPI

### PR DESCRIPTION
**Pull request by TroyDota**

### CODE REVIEW

**Review by AnatoleAM — CHANGES_REQUESTED** [review]
image

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,86 @@
...
-SOFTWARE.
+### “Commons Clause” License Condition v1.0
+
+The Software is provided to you by the Licensor under the License, as defined below, subject to the following condition.
+
+Without limiting other conditions in the License, the grant of rights under the License will not include, and the License does not grant to you,  right to Sell the Software.
+
+For purposes of the foregoing, “Sell” means practicing any or all of the rights granted to you under the License to provide to third parties, for a fee or other consideration (including without limitation fees for hosting or consulting/ support services related to the Software), a product or service whose value derives, entirely or substantially, from the functionality of the Software.  Any license notice or attribution required by the License must also include this Commons Cause License Condition notice.
+
+**Software:** 7TV EventsAPI
```
[the remark this answers is not in the document]
Comment (AnatoleAM) [reply LICENSE:9]: Also file name should be `LICENSE.md` so it gets markdown on web view

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -34,3 +34,32 @@ jobs:
           tags: |
             harbor.disembark.dev/7tv/event-api:${{ github.sha }}
             harbor.disembark.dev/7tv/event-api:latest
+
+  deploy:
+    name: Deploy (Production)
+    runs-on: ubuntu-latest
```
Comment (AnatoleAM) [inline .github/workflows/build.yaml:40]: Running on shared runners - would this cause issues with the new internal harbor thing?

Comment (TroyDota) [reply .github/workflows/build.yaml:40]: no its only a k8s deploy

File: src/events/event-stream.go [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package events
```
Comment (AnatoleAM) [inline src/events/event-stream.go:1]: I think we could group more of the logic together and do something prettier but considering we only have one route for now it's fine. Seems good to me

**Review by AnatoleAM — APPROVED** [review]

**Pull request merged**